components/dashboard/stats-card.tsx

```
@@ -40,46 +52,69 @@
+              id={`${title.toLowerCase().replace(/\s+/g, '-')}-description`}
+              className={DESCRIPTION_STYLES}
+            >
               {description}
             </p>
           )}
           {trend && (
-            <div className="mt-2 flex items-center space-x-1">
+            <div 
+              className="mt-2 flex items-center space-x-1"
+              aria-label={trendDescription}
+            >
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)